templates/terms_and_conditions.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="Paper-copy">
+      <p>NurseConnect is a National Department of Health (NDoH) initiative. It is the one stop shop where nurses can find information, ask for expert help, receive emotional support and have a line of direct communication with the National Department of Health. It utilises mobile software to give nurses in South Africa access to clinical, psychosocial and learning content. </p>
```

Comment:
This indentation is much bigger than the other indentations in this file. Sorry to be nitpicky, but it would be nice if all the indentations were the same.

We also probably don't need to wrap this in a div, we're not applying any specific styles to it, and it's already wrapped in a div by the base.html template

Reply:
okay will change that